examples/quickstart/custom_serialization.cpp

```diff
@@ -0,0 +1,189 @@
+//  Copyright (c) 2022 John Sorial
+//
+//  SPDX-License-Identifier: BSL-1.0
+//  Distributed under the Boost Software License, Version 1.0. (See accompanying
+//  file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+// This example is meant for inclusion in the documentation.
+
+#include <hpx/format.hpp>
+#include <hpx/future.hpp>
+#include <hpx/hpx_main.hpp>
+#include <hpx/include/actions.hpp>
+#include <hpx/include/runtime.hpp>
+#include <hpx/serialization.hpp>
+#include <iostream>
+
+//[PointMemberSerialization
+struct PointMemberSerialization
+{
+    int x{0};
+    int y{0};
+
+    // Required when defining private fields
+    friend class hpx::serialization::access;    // Provide member access to HPX
+
+    template <typename Archive>
+    void serialize(Archive& ar, const unsigned int)
+    {
+        ar& x& y;
```

Comment:
We usually write it as:
```suggestion
        // clang-format off
        ar & x & y;
        // clang-format on
```